config/global.ini.php

```diff
@@ -1095,6 +1095,7 @@
 type = BASIC ; proxy type for outbound/outgoing connections; currently, only BASIC is supported
 host = ; Proxy host: the host name of your proxy server (mandatory)
 port = ; Proxy port: the port that the proxy server listens to. There is no standard default, but 80, 1080, 3128, and 8080 are popular
+exclude = ; Hosts to exclude from proxy: optional. Works only with libcurl and sets the no_proxy option
```

Comment:
Might be good to specify what value we expect here. like `Comma separated list of hosts to exclude...`

Reply:
Changed text to mention Comman separated list of hosts and removed the limitation to curl as it is now for all transports